### ISSUE TITLE
[issue] - Background apps list not showing

### ISSUE THREAD

**Issue by RastaTaz** [falzonv/discreet-launcher#229]
Since few releases, background apps list doesn't work, either with menu (when hitting "square") or gestures (sweep from screen bottom).

=> I'm currently using v6.3.0 but had bug with previous version (can't tell which one) on a Ulephone X7Pro running Android 10.

- when hitting "square" screen flickers then gets back to running app
- when using sweep gesture, I get sent back to launcher
- when configuring stock launcher issue gets solved
- when clearing cache and config of app, issue still persist

See video uploaded on link below: I first show with square with stock launcher then with discreet launcher (don't pay attention on last 25 seconds: I was searching how to stop screen recording...)
https://user-images.githubusercontent.com/28950854/175789374-02a08b7a-bc59-498b-88ad-9ffee523e8c8.mp4

**Comment by falzonv** [comment]
Hello,

Thank you very much for the report and taking the time to make a video!

It is indeed a very strange bug, using an Android 10 emulator I am not able to reproduce it (that is, the square button works fine and as expected) so maybe this is a device-specific bug...

Searching on Internet, I found the following Reddit post about the exact same phone:
https://www.reddit.com/r/AndroidQuestions/comments/iz45j1/any_launcher_that_has_a_working_recents_on/

> *Any launcher that has a working recents on Android 10 (Ulefone Armor X7)?
> [...]
> I've tried like 10 different launchers, and only "quickstep", which is the built-in launcher, can act as a recents provider.
> [...]
> I put up another thread, some people suggested some launchers, none of them work for recent apps.*

=> Do you observe the same bug with other custom launchers?
If you know about a custom launcher with which "Recent apps" works as expected, and if it is open source, I could have a look and search how the developer managed to solve this.

Globally, the options from my side are quite limited anyway:
- As far as I know, I can only customize the "Back" button (triangle), which I did to close the favorites or the drawer when appropriate. The "Home" and "Recent apps" buttons are protected, probably to prevent malwares misusing them.
- The "Recent apps" screen/list is also managed by the system, Discreet Launcher doesn't even have access to it.

Best regards.

**Comment by RastaTaz** [comment]
Hi Vincent,
Thanks for your answer and extra research.

Can't tell if issue happens with other launchers but as far as I remember I didn't spotted anything with few others I tried before... and I don't have another one right now to check - as IMHO discreet is the only one worth having ;).
All I can say for the moment is that it was working with previous versions of discreet launcher and I did't have any update from vendor since (but with google play services you never know...).

=> I'll have a look into this with the new perspective you bring to the subject and get back to you in case I have any news worth sharing on this topic, but I'll close the issue for the time being.

Thanks again and keep that great work you provided to the open source community !
